Source/ASPagerFlowLayout.m

```diff
@@ -20,6 +20,7 @@ @interface ASPagerFlowLayout () {
 
 @end
 
+//TODO make this an ASCollectionViewLayout
```

Comment:
This, especially when combined with vertical flow layouts inside the pages, will be a truly revolutionary impact on performance in pin-open and pin-to-pin swiping performance.